module/mfrc522_module.h

```diff
@@ -8,11 +8,17 @@
 #include <linux/types.h>
 #include <linux/miscdevice.h>
 
+struct mfrc522_stats {
```

Comment:
```suggestion
struct mfrc522_statistics {
```
`stats` is a bit too close to `mfrc522_state` imo